frontend/src/components/NavBar/index.jsx

```diff
@@ -66,28 +76,35 @@ function NavBar() {
                 <Link to={"contentcreator"} className="w-full h-[5%] flex items-center">
                     Content Creator Tool
                 </Link>
-                <div className="w-full h-[5%] flex items-center mt-auto text-lg justify-evenly cursor-pointer" 
-                    onClick={()=>{changeTheme(theme === "dark" ? "light":"dark");console.log(theme)}}>
+                <button className="w-full h-[5%] flex items-center mt-auto text-lg justify-evenly cursor-pointer" onClick={handleLogout}>
+                    Logout
+                </button>
+
+                <button className="w-full h-[5%] flex items-center mt-auto text-lg justify-evenly cursor-pointer" onClick={() => changeTheme(theme === "dark" ? "light" : "dark")}>
```

Comment:
@fungame2270 Check if the functionality or aspect changed in any way shape or form please.